frontend/admin/.eslintrc

```diff
@@ -35,9 +35,16 @@
         "prettier",
         "react",
         "react-hooks",
-        "@typescript-eslint"
+        "@typescript-eslint",
+        "formatjs"
     ],
     "rules": {
+        "formatjs/enforce-id": [
```

Comment:
This is showing up as missing in the linting of a merge commit. Does something else need to be added to enable it?

```
 1:1   error    Definition for rule 'formatjs/enforce-id' was not found  formatjs/enforce-id
```

Is this needed? https://formatjs.io/docs/tooling/linter/

Reply:
Oh, good catch!  It wasn't coming up in the vs code problems pane but I see it now.  I added the missing plugin and fixed a few more configuration issues I noticed.